entries/o/o2switch.fr.json

```diff
@@ -0,0 +1,14 @@
+{
+    "o2switch": {
+        "domain":"o2switch.fr",
+        "url": "https://clients.o2switch.fr",
+        "tfa": [
+            "email"
+        ],
+        "keywords": [
+            "hosting",
+            "domains"
+        ],
+        "notes": "Email 2FA is always active, when logging in from a new device."

```

Comment:
Is 2FA only when logging from a new device, or is it every time you log in?

Reply:
As I noticed, when logging from a new device, but some times when I log back